README.md

```diff
@@ -89,6 +89,26 @@ TIME=true HTML=true APP_ENV=test bundle exec rake test
 ```
 This generates a report with test times. You can find the report in optimizer-api/test/html_reports folder.
 
+
+If you want to run a specific test file (let's say real_cases_scheduling_test.rb file only):
+```
+APP_ENV=test bundle exec rake test TEST=test/real_cases_scheduling_test.rb
+```
+If you want to run only one specific test (let's say test_instance_clustered test only) you can use `focus` or call:
+```
+APP_ENV=test bundle exec rake test TESTOPTS="--name=test_instance_clustered"
+```
+Tests are splitted into several sets and you can replay only one or more set(s):
+```
+APP_ENV=test bundle exec rake test:api
+APP_ENV=test bundle exec rake test:api test:models
+```
+You can list all available tasks with:
+```
+bundle exec rake -T
```

Comment:
`rake -T` only shows the tasks which has a description so currently this doesn't show the sub test tasks because they are defined in an unexpected format. So currently it shows 

```
rake resque:failures:sort  # Sort the 'failed' queue for the redis_multi_queue failure backend
rake resque:work           # Start a Resque worker
rake resque:workers        # Start multiple Resque workers
rake test                  # Run tests
rake test_dump_vrp         # Dump all necessary fixture files
```

With the modification I suggested below, it shows all the test options

```
rake resque:failures:sort  # Sort the 'failed' queue for the redis_multi_queue failure backend
rake resque:work           # Start a Resque worker
rake resque:workers        # Start multiple Resque workers
rake test                  # Run tests
rake test:api              # Run tests for api
rake test:clustering       # Run tests for clustering
rake test:lib              # Run tests for lib
rake test:model            # Run tests for model
rake test:periodic         # Run tests for periodic
rake test:structure        # Run tests for structure
rake test_dump_vrp         # Dump all necessary fixture files
```

Reply:
what is your suggestion instead of `bundle exec rake -T`

Reply:
with my suggestion below, it works as expected  -- i.e., it lists all the tests.